test/functional/testCases.js

```diff
@@ -0,0 +1,1641 @@
+const config = require('./config-test.js');
+var chai = require('chai');
+var chaiMatchPattern = require('chai-match-pattern');
+chai.use(chaiMatchPattern);
+var _ = chaiMatchPattern.getLodashModule();
+chai.config.truncateThreshold = 0;
+
+const globalEnv = {
+    service: 'smartgondor',
+    servicePath: '/gardens',
+    apikey: '123456',
+    entity_type: 'TestType',
+    entity_name: 'TestType:TestDevice',
+    deviceId: 'TestDevice'
+};
+
+const testCases = [
+    // BASIC TESTS
+    {
+        describeName: '0010 - Simple group without attributes',
```

Comment:
I'd suggest a similar apporach than in CB test cases.

* The number identifies the issue number associated to the test
* For those tests not associated to specific issue (as the "starting ones"), let's use 0000